packages/vue/src/examples/pages/layout/Layout.stories.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import Layout from "./Layout";
```

Comment:
Layout?

Reply:
@pspaczek we already have stories for header and footer, you can just include this as a story  of megamenu (with header)

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/7ddf887974cc3400e66d8fd25edcb597ff9f6e4a

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/7ddf887974cc3400e66d8fd25edcb597ff9f6e4a

Reply:
include SfHeader and SfMegaMenu: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/99037386f6d8d4469a48ea18902fe01480ba5e04